docs/metrics/tp-fp-fn-tn.md

```diff
@@ -0,0 +1,194 @@
+
+For multiclass classification tasks, these four metrics are defined **per class**. This technique,
+also known as **one-vs-rest** (OvR), essentially evaluates each class as a binary classification problem.
+
+Consider a classification problem where a given image belongs to either the `Airplane`, `Boat`, or `Car` class. Each of
+these TP / FP / FN / TN metrics is computed for each class. For class `Airplane`, the metrics are defined as follows:
+
+| Metric | Example |
+| --- | --- |
+| True Positive | Any image predicted as an `Airplane` that is labeled as an `Airplane` |
+| False Positive | Any image predicted as an `Airplane` that is _not_ labeled as an `Airplane` (e.g. labeled as `Boat` but predicted as `Airplane`) |
+| <nobr>False Negative</nobr> | Any image _not_ predicted as an `Airplane` that is labeled as an `Airplane` (e.g. labeled as `Airplane` but predicted as `Car`) |
+| True Negative | Any image _not_ predicted as an `Airplane` that is _not_ labeled as an `Airplane` (e.g. labeled as `Boat` but predicted as `Boat` or `Car`) |
+
+#### Multi-label
+
+In a **multi-label** classification task, TP / FP / FN / TN are computed per class, like in multiclass classification.
+
+A sample is considered to be a positive one if the ground truth **includes** the evaluating class; otherwise, it’s a
+negative sample. The same logic can be applied to the predictions, so, for example, if a classifier predicts that this
+sample belongs to class `Airplane` and `Boat`, and the ground truth for the same sample is only class `Airplane`, then
+this sample is considered to be a TP for class `Airplane`, and FP for class `Boat`.
+
+Multi-label classification tasks can alternately be thought of as a collection of binary classification tasks.
+
+### Object Detection
+
+There are some differences in how these four metrics work for a detection task compared to a classification task.
+Rather than being computed at the sample level (e.g. per image), they're computed at the instance level (i.e. per object)
+for instances that the model is detecting. When given an image with multiple objects, each inference and each ground truth
+is assigned to one group, and the definitions of the terms are slightly altered:
+
+| Metric | | Description |
+| --- | --- | --- |
+| True Positive | TP | Inference matched with a ground truth |
+| False Positive | FP | Inference that is not matched with a ground truth |
+| <nobr>False Negative</nobr> | FN | Ground truth that is not matched with an inference |
+| True Negative | TN | <p>:kolena-warning-sign-16: **Poorly defined for object detection!** :kolena-warning-sign-16:</p><p>In object detection tasks, a true negative is any non-object that isn't detected as an object. This isn't well defined and as such true negative isn't a commonly used metric in object detection.</p><div>Occasionally, for object detection tasks "true negative" is used to refer to any image that does not have any true positive or false positive inferences.</div> |
```

Comment:
`kolena-warning-sign` doesn't render for me when I built it locally.

<img width="649" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/75b52b3e-6d93-488c-8910-b65e971e3778">

Reply:
Have you restarted your docs server?

Reply:
No, I haven't!  It works now 👍 
<img width="681" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/70b3d19c-fe23-484e-8f48-65ef2eef71ea">